bazel/third_party/openexr_deps.bzl

```diff
@@ -9,16 +9,12 @@ load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
 def openexr_deps():
     """Fetches dependencies (zlib and Imath) of OpenEXR and Skylib for header generation."""
 
-    maybe(
-        http_archive,
-        name = "net_zlib_zlib",
-        build_file = "@com_openexr//:bazel/third_party/zlib.BUILD",
-        sha256 = "b3a24de97a8fdbc835b9833169501030b8977031bcb54b3b3ac13740f846ab30",
-        strip_prefix = "zlib-1.2.13",
-        urls = [
-            "https://mirror.bazel.build/zlib.net/zlib-1.2.13.tar.gz",
-            "https://zlib.net/zlib-1.2.13.tar.gz",
-        ],
+    http_archive(
```

Comment:
use `maybe` again:

```
maybe(
        http_archive, 
        name = "libdeflate",
        build_file = "@com_openexr//:bazel/third_party/libdeflate.BUILD",
        sha256 = "225d982bcaf553221c76726358d2ea139bb34913180b20823c782cede060affd",
        strip_prefix = "libdeflate-1.18",
        urls = ["https://github.com/ebiggers/libdeflate/archive/refs/tags/v1.18.tar.gz"],
)
```

The whole point of `maybe` is that it checks if a dependency with the name `libdeflate` was already added - and if so it skips it - can happen in large project that use the same transitive dependencies

Reply:
yeah, I debated removing the maybe, but not being a bazelite, I didn't know if maybe gave it a chance to be overridden by the user (much like is possible in cmake where I can specify the (potentially local) repo...

Reply:
i think the maybe is now consistent with the others. Where I was embedding before, there wasn't a choice of maybe given the hijinks, now that it is a normal library, it's fine